config/environments/test.rb

```diff
@@ -4,26 +4,29 @@
 Argo::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb
 
-  config.assets.compress = Settings.APPLICATION.ASSETS.COMPRESS
+  # The test environment is used exclusively to run your application's
```

Comment:
I kept it this way to be more inline with what the Rails default does: https://github.com/rails/rails/blob/v4.2.4/railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

That way anyone stubbling upon could compare and quickly see, no special config thing.